jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/DocumentSummary.java

```diff
@@ -24,6 +24,8 @@
 public class DocumentSummary extends GenericSummary<String> {
 
     private String documentId;
+    private String documentServerTemplateId;
+    private String documentIdentifier;
```

Comment:
the documentId is being used in ProcessDocumentListViewImpl and I am not sure if we can use the documentIdentifier for it instead without breaking things. If so I'd be happy to remove it. Let me know. thanks